scss/styles.scss

```diff
@@ -95,11 +97,49 @@
 .nav-title {
   align-self: center;
   font-size: $font-big;
-  padding-right: 25px;
 }
 
 .nav-logo {
   align-self: center;
-  padding-left: 25px;
   width: 200px;
 }
+
+.footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  height: auto;
+  justify-content: center;
+  left: 0;
+  padding: 50px;
+  position: absolute;
```

Comment:
this is not needed, you should alter your html to achieve this. change the html to this:
```
<body>
  <nav>
    ...
  </nav>
  <div class="body-container">
    ...
  </div>
  <footer>
    ...
  </footer>
</body>
```